validator/iam_required_bindings_test.rego

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GCPIAMRequiredBindingsConstraintV1
+
+import data.test.fixtures.iam_required_bindings.assets as fixture_assets
+import data.test.fixtures.iam_required_bindings.constraints as fixture_constraints
+
+# Find all violations on our test cases
+find_violations[violation] {
```

Comment:
I think this might be able to leverage some of test_utils.rego

Reply:
Have changed- PTAL.

Reply:
I think you can also remove find_violations

Reply:
Thanks for the catch! Have fixed.